fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -50,19 +55,19 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
 
         <RadioGroup
           id="job-category-suggest-select"
-          onChange={(event) => handleChoiceSelect(event.currentTarget.value)}
```

Comment:
I actually messed this up in the previous PR; it wasn't caught because of the `any` that is now properly typed